### ISSUE TITLE
Font color should be black, not grey

### ISSUE THREAD

**Issue by idontusenumbers** [a85/POSTMan-Chrome-Extension#2450]
Font color should be black, not grey. Grey-on-white is hard to read.

**Comment by sdnts** [comment]
Hey @idontusenumbers, which part of the app do you feel is hard to read? We'll try to add more contrast to places where it is hard to read.

**Comment by idontusenumbers** [comment]
![postman](https://cloud.githubusercontent.com/assets/1175031/19953804/3f10f38e-a140-11e6-9e36-2cd3610fdaa1.png)

**Comment by vkaegis** [comment]
Hey @idontusenumbers, thanks for reporting this! We're definitely looking into accessibility problems with our UI and will be looking to improve on the contrast of content throughout the app in the coming updates.

Some of the greys are there by choice, to reduce the visual stress of an interface with a lot of visible elements. That being said, at some time in the future we definitely hope to have a completely customizable interface where users can style it however they chose to.

**Comment by idontusenumbers** [comment]
Visual stress is CREATED by the grey text. It is very hard to read. Contrast should be increased.

**Comment by connorsadlervelo** [comment]
Is there any way to change this in a .css file or similar, please?
My eyes hurt ;]

**Comment by sivcan** [comment]
@connorsadlervelo - Have you tried using the dark theme ? 
Does that help for now ?

If you haven't then: 
Go to Settings and change it 
![image](https://user-images.githubusercontent.com/12631777/36780823-181d26a4-1c9a-11e8-8ebe-90b0fe40f6f8.png)

**Comment by connorsadlervelo** [comment]
Thanks for that - I stumbled on it myself and it's much better on the eyes
Thanks again!

**Comment by sivcan** [comment]
@connorsadlervelo - Awesome! :D

**Comment by i-like-robots** [comment]
The app remains very low contrast which can make it hard to use even for users without eyesight problems. Is this issue still on anybody's to-do list?

I'm currently working with an average screen near a window and I am unable to use the app with either theme due to the low contrast of the text.

**Comment by xeruf** [comment]
Same, see https://github.com/postmanlabs/postman-app-support/issues/3843

**Comment by scharf** [comment]
I cannot switch to dark mode because it confuses me even more. The low contrast makes it **very** hard to work with the app.

**Comment by nvncbl** [comment]
Dark mode (color) was an incorrect response to the problem (contrast).  I feel like the aesthetic is attempting to be as "soft" and as "inoffensive" as possible.  Of all the windows/apps open on my three monitors, Postman is extremely washed out, as if there's a 0.7 opacity filter applied, which greatly detracts from usability.

**Comment by scharf** [comment]
Applications like Evernote used to have this type of low contrast UI but they changed it to normal contrast.

**Comment by christophe-riolo** [comment]
I came to report similar difficulty to read notably the collections, the contrast is too low and there is no way to at least change the font size (we can change the editor font size but it has no effect on the sidebar), which would at least help compensate a bit.

**Comment by dsaddan** [comment]
Please please fix this issue. It has been going on for over 4 years. The tool is great, the features are great, the font color is not.

**Comment by jwfx** [comment]
Another year has passed without any improvement in this area. This application is amazing feature-wise, but the UI in its current state is really eye-cancer inducing. Switching to dark mode is also not an option if you are not a regular user of dark mode, that said, it also suffers from the same low-contrast, small font problem that the regular theme does.

Could 2022 please be the year where eye-cancer goes away?

**Comment by SamSoldatenko** [comment]
? Will you finally fix the colours?

**Comment by chris-torruella** [comment]
Hey everyone, we heard you and I'm excited to share that Postman v11 introduces a whole set of application themes accessible via Settings/Themes. Now you can also sync these with your System preferences. But very importantly, we've also included High Contrast themes for enhanced contrast as needed. We hope these will help to elevate your workflow, and we're eager to hear your thoughts. Cheers!
